test/tests/parallel/set3/dovecot_inet_protocol.bats

```diff
@@ -0,0 +1,47 @@
+TEST_NAME_PREFIX='Dovecot protocols:'
+CONTAINER_NAME='dummy'
+CONTAINER_NAME_ALL='dms-test-dovecot_protocols_all'
+CONTAINER_NAME_IPV4='dms-test-dovecot_protocols_ipv4'
+CONTAINER_NAME_IPV6='dms-test-dovecot_protocols_ipv6'
```

Comment:
Are the individual `@test` tests running concurrently? Or sequentially?

- The `dummy` usage could probably be avoided.
- `CONTAINER_NAME_*` can use `export` keyword to be available throughout execution of the file AFAIK? If so you could move these into `setup_file()` co-located with the relevant block just before `CONTAINER_NAME`.

Reply:
Inside a test file, test run sequentially, IIRC. The `dummy` can be avoided, right. I would like to avoid using `export` in `setup_file` as I want to have all variables that need to be set for a file to be set in the very beginning.

Reply:
> I would like to avoid using `export` in `setup_file` as I want to have all variables that need to be set for a file to be set in the very beginning.

I prefer co-location myself, especially since it avoids the `dummy` value in this case which would make any additional tests pass the check for `CONTAINER_NAME` in `setup.bash` if the `dummy` usage became common place for scenarios like this one.

`setup_file()` is fairly close to setting at the beginning :sweat_smile: 

Not going to block on this though :+1:

Reply:
I agree, setting it in `setup_file` would be okay as well, but sourcing `setup.bash` alone is enough for the check, and since we run `setup_file` after we source `setup.bash`, I think there is little room to put the ENV other than at the beginning of the file.